docs/release/trg-0/trg-1-07.md

```diff
@@ -0,0 +1,30 @@
+---
+title: TRG 1.07 - End-User Manual (User Assistance)
+sidebar_position: 1
+---
+
+:::caution
+Proposed release date: 15th of January 2024
+:::
+
+| Status     | Created      | Post-History                           |
+|------------|--------------|----------------------------------------|
+| Draft      | 31-Jan-2024  | created initial Draft                  |
+
+## Why
+
+End-User Manual is essential for providing user assistance and ensuring a positive user experience. The user of the application is the receipient of this documentation. Typical contents are screenshots, description of functionality and steps that guide through critical processes.
+
+## Description
+
+- The End-User Manual **has to** be stored in the folder: ```docs/user/```
+- End-User Manual is available with the software at the same time.
+- End-User Manual is correct and up to date; English is a must.
+- The documentation is of appropriate maturity to be handed over to any Operations Company with global business practice intentions.
+- In Markdown format. Link to the document available. Must point to the leading repository within Tractus-X.
+
+## Best Practice
```

Comment:
Also same as on the other files. Define actual best practices and not how to engage in a process

Reply:
delete lines 28 to end.
new text for best practices (bulletpoints):
The Enduser manual is in sync with the latest released version of the component. It is checked and updated, once an up-revision is foreseen.
The content is of appropriate maturity, so that any Operations Company with global business practice intentions is invited to use and build on it.
Anchor a link to your Enduser manual within your readme.md (see TRG 1.01; https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-1)